protobufs/livekit_sip.proto

```diff
@@ -453,7 +473,7 @@ message SIPUri {
   string user = 1;
   string host = 2;
   string ip = 3;
-  string port = 4;
+  uint32 port = 4;
```

Comment:
@biglittlebigben Fixing a typo here. Hope it's still okay to do so.

Reply:
is this an existing field that's already used somewhere? this might be a problem right?

Reply:
Yes, it's a breaking change, but this field is not used anywhere yet.